homeassistant/components/alexa/capabilities.py

```diff
@@ -66,6 +73,51 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def get_resource_by_unit_of_measurement(entity: State) -> str:
+    """Translate the unit of measurement to an Alexa Global Catalog keyword."""
+    if not (unit := entity.attributes.get("unit_of_measurement")):
+        return AlexaGlobalCatalog.SETTING_PRESET
+
+    if unit == UnitOfTemperature.CELSIUS:
```

Comment:
Let's make this a mapping instead of a big if tree :)

Reply:
Working on it